packages/office-ui-fabric-react/src/components/Callout/Callout.scss

```diff
@@ -25,6 +25,7 @@ $Callout-smallbeak-slant-width: 16px;
   @include ms-drop-shadow(0,0, 5px, 0px);
   position: absolute;
   box-sizing: border-box;
+  border: 1px solid $ms-color-neutralLight;
```

Comment:
As much as I want this change to happen, this isn't the right PR for it and I'd recommend splitting it out into it's own PR in an effort to keep PRs Single Responsibility.